lambda/shared/templates/delete-integration-submitted-bceid/template.html

```diff
@@ -0,0 +1,19 @@
+<h1>Hello IDIM Team,</h1>
+<p>We are notifying you that we have received a request to delete an integration. The request details are below:</p>
+<ul>
+  <li><strong>Requester: </strong>{{integration.requester}}</li>
+  <li><strong>Project name: </strong>{{integration.projectName}} #({{integration.id}})</li>
+  <li><strong>Accountable person: </strong>{{integration.accountableEntity}}</li>
+  <li>
+    <strong>URIs:</strong>
+    <ul>
+      {{integration.devUris}} {{integration.testUris}} {{integration.prodUris}}
```

Comment:
Will this display the urls all in a line or did we want them to be on 3 lines?

Reply:
it includes `li` elements so it will be displayed one line at a time
```
 return `<li> ${prefix}: ${uris.join(', ')} </li>`;
```

Reply:
That makes sense